### DISCUSSION TITLE
Form loses its values on orientation change

### DISCUSSION THREAD

**Discussion by alex-clear** [react-hook-form/react-hook-form#9870] [General]
Hi
We faced with very strange behaviour of react form when we tested it with mobiles
E.g. we have very simple add phone form and when we flip orientation the form get reset and all values are gone
Such issue we have in all our react forms and we cannot find any solution
Anyone faced with something similar and can provide some tip to resolve this one?

https://user-images.githubusercontent.com/97834715/216819547-b6e6c705-5555-40c2-aebf-a708b58a3e42.mov

**Comment by Moshyfawn** [comment]
Please, provide a minimal reproduction sandbox instead. A cropped out video doesn't help us debug the issue
